rule-packs/sophos-endpoint-security

```diff
@@ -0,0 +1,166 @@
+#start-sophos-endpoint-misconfigurations
+[
+  {
+    "name": "sophos-misconfigured-endpoints-1",
+    "description": "This will query for endpoints that do not have sophos edr installed.",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "FIND Device !PROTECTS << sophos_endpoint",
+        "version": "v1"
+      }
+    ]
+    "alertLevel": "MEDIUM"
+  },
+  {
+    "name": "sophos-misconfigured-endpoints-2",
+    "description": "This will query for endpoints that are missing a relationship with a user. This will help determine potential rogue devices",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "Find sophos_sensor that !HAS User THAT IS Person THAT IS okta_user|azure_user|google_user",
```

Comment:
is it 'sophos_device', 'sophos_endpoint' or 'sophos_sensor'? I think a mix of these is used in the rules and all need to match

Reply:
sophos endpoint is the sensor
sophos device is the device